pump/urls.py

```diff
@@ -16,7 +15,7 @@
 site_media_root = os.path.join(os.path.dirname(__file__), "../media")
 
 urlpatterns = [
-    url(r'^accounts/', include('django.contrib.auth.urls')),
+    path("accounts/", include('django.contrib.auth.urls')),
```

Comment:
I think this should use re_path
`re_path(r'^accounts/', include('django.contrib.auth.urls')),`

Reply:
Change made

Reply:
I think the re_path is necessary when doing more elaborate stuff, like for example:
```
    re_path(r'^pageblock/jsonimport/(?P<pageblock_id>\d+)/$',
            import_pageblock_json, {}, "import-pageblock-json"),
```

In this case, `path("accounts/"` will match to the same thing as the regex `r'^accounts/$'` I think. But, either one will work.